components/AdminProfileForm/AdminProfileForm.module.css

```diff
@@ -0,0 +1,104 @@
+
+.page {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin: 7%;
+    margin-top: 5%;
+    width:100%;
+    height: 100%;
+    background-color: #FAFAFA 100vh;
+}
+
+.pagetitle {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 1%;
+}
+
+.namebar {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 1%;
+    margin-bottom: 2%;
+}
+
+.boxes {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    margin-top: 2%;
+    gap: 5%;
+}
+
+.box {
+    padding: 4%;
+    justify-content: space-between;
+    align-items: center;
+    border: 2px solid var(--sapphire-light);
+    border-radius: 25px;
+    width: 100%;
+    background-color: white;
+}
+
+.hr {
+    height: 2px;
+    color: var(--sapphire-light);
+}
+
+.info {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-right: 5%;
+}
+
+.fields {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+}
+
+.editable-field {
+    outline: none;
+    text-align: right;
+    border: none;
+    font-size: 13px;
+}
+
+.edit-buttons-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: flex-start;
+    width: 20%;
+}
+
+.editable-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    width: 30%;
+}
+
+.editable-field {
```

Comment:
for the editable fields, when you press "edit" is it possible to still make all the fields technically editable at once, but have hover states on them to indicate which field is currently in focused state? my designs in figma model off notion where u can technically edit all the fields at once by clicking into any field, but the states differ in that it highlights grey when u hover over the field, and turns into a white box + drop shadow once you're ac editing. i think this just makes it more intuitive to see what ur editing at a given time, as well as a subtle confirmation once u click out of that focused state.
if this is what u were talking about being too hard to implement tho, dw about it, i might change how the design looks then bc i dont like how it looks w all drop shadows being activated at once.

https://user-images.githubusercontent.com/77181350/147707314-6cd0a420-540a-4a61-96f4-8180696abc70.mov